extensions/resource-deployment/src/main.ts

```diff
@@ -6,15 +6,40 @@
 
 import vscode = require('vscode');
 import { ResourceDeploymentDialog } from './ui/resourceDeploymentDialog';
+import { ToolsService } from './services/toolsService';
+import { NotebookService } from './services/notebookService';
+import { ResourceTypeService, PackageJsonPath } from './services/resourceTypeService';
 
 export function activate(context: vscode.ExtensionContext) {
+	const toolsService = new ToolsService();
```

Comment:
I need to investigate this more to see what we get by default - but you should check with Aditya for what telemetry we currently get for extensions (if any). For example a couple key ones that we should be interested in are extension activations and command executions - but both of these should be possible to be made generically available at a higher level so maybe that work is already done.

(and in addition think if there's any other telemetry you'd like to add)

Reply:
I added the extension related telemetry, extension activation and command usages are handled by ADS core. I will add telemetry for notebook launch later.